pkg/installer/custominstallconfig.go

```diff
@@ -102,15 +106,20 @@ func (m *manager) applyInstallConfigCustomisations(installConfig *installconfig.
 		}
 	}
 
-	// Add ARO Manifests to bootstrap Files and CVO Overrides
 	if aroManifestsExist {
+		// Add ARO Manifests to CVO Overrides
 		if err = appendFilesToCvoOverrides(aroManifests, g); err != nil {
 			return nil, err
 		}
 
+		// Add ARO Manifests to bootstrap Files
 		if err = appendFilesToBootstrap(aroManifests, g); err != nil {
 			return nil, err
 		}
+		// Update Master Pointer Ignition with ARO API-Int IP
+		if err = replacePointerIgnition(aroManifests, g, dnsConfig); err != nil {
```

Comment:
`aroManifestsExist` is set if Hive has supplied extra manifests. I don't think we want this to be conditional on that.